### PR TITLE
Added FAQ "How big is the Monero blockchain?"

### PR DESCRIPTION
**Pull request by tonyern**
I added a new FAQ under the "Node and Wallet" section. I have the question and answer in English and Spanish.

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -430,6 +430,8 @@ faq:
   anodetor: Support for Tor is still in its infancies, but it's already possible to natively send transactions through the network and to run a Monero @daemon on the Tor network. Better Tor and I2P integrations are in progress.
   qfullpruned: How do I decide if I should run a full node or a pruned node?
   afullpruned: "A full @node requires a considerable amount of storage and could take a long time to download and verify the entire blockchain, especially on older hardware. If you have limited storage, a pruned node is recommended. It only stores 1/8th of unnecessary blockchain data while keeping the full transaction history. If plenty of storage is available, a full node is recommended but a pruned node still greatly contributes to the network and improves your privacy."
+  qblocksize: How big is the Monero blockchain?
+  ablocksize: "The Monero @blockchain is always growing so there is no fixed size. As of 2021, the blockchain is around 95-100GB. "
```
Comment (erciccione) [inline _i18n/en.yml:434]: Better add the size of a pruned blockchain as well.

File: _i18n/es.yml [hunk 2]
```diff
@@ -730,6 +730,7 @@ faq:
     En este sitio web hay una lista disponible de casas de cambio donde es posible
     comprar/vender Monero (XMR):'
   qavoidbc: ¿Puedo evitar descargar toda la cadena de bloques?
+  qblocksize: ¿Qué tan grande es la cadena de bloques Monero?
```
Comment (erciccione) [inline _i18n/es.yml:733]: Translations must be submitted to weblate: translate.getmonero.org

Please revert all changes to this file.

**Review by erciccione — CHANGES_REQUESTED** [review]
I proposed some fixes.

File: _i18n/en.yml [hunk 3]
```diff
@@ -430,6 +430,8 @@ faq:
   anodetor: Support for Tor is still in its infancies, but it's already possible to natively send transactions through the network and to run a Monero @daemon on the Tor network. Better Tor and I2P integrations are in progress.
   qfullpruned: How do I decide if I should run a full node or a pruned node?
   afullpruned: "A full @node requires a considerable amount of storage and could take a long time to download and verify the entire blockchain, especially on older hardware. If you have limited storage, a pruned node is recommended. It only stores 1/8th of unnecessary blockchain data while keeping the full transaction history. If plenty of storage is available, a full node is recommended but a pruned node still greatly contributes to the network and improves your privacy."
+  qblocksize: How big is the Monero blockchain?
+  ablocksize: "The Monero @blockchain is always growing so there is no fixed size. As of 2021, the full-node blockchain is around 95-100GB. Pruned-node blockchain is 30GB. Check out Moneropedia entry @pruning to learn the difference between full-node and pruned-node blockchains."
```
Comment (erciccione) [inline _i18n/en.yml:434]:
```suggestion
  ablocksize: "The Monero @blockchain is always growing so there is no fixed size. As of 2021, the full blockchain is around 95-100GB. A pruned blockchain is about 30GB. Check out Moneropedia entry @pruning to learn the difference between full-node and pruned-node blockchains."
```

**Review by erciccione — APPROVED** [review]
Looks good now. Thanks :)

Closes #1371

**Comment by tonyern** [comment]
What happens after approval of the pull request? Do I have to do anything else?

**Review by rehrar — APPROVED** [review]

**Comment by rehrar** [comment]
The only comment I'd make here is "as of 2021" since late 2021 the blockchain will be quite a different size. But it's not super important atm.

**Pull request merged**